motionplan/errors_test.go

```diff
@@ -0,0 +1,34 @@
+package motionplan
+
+import (
+	"testing"
+
+	"github.com/pkg/errors"
+	"go.viam.com/test"
+)
+
+func TestNewIKConstraintErr(t *testing.T) {
+	constraintFailCnt := 10000
+
+	t.Run("some errors exceed cutoff", func(t *testing.T) {
+		failures := map[string]int{
+			"obstacle constraint: violation between ur5e-modular:forearm_link and box1 geometries":                  2913,
+			"self-collision constraint: violation between gripper-1:clamp and ur5e-modular:wrist_1_link geometries": 97,
+		}
+
+		//nolint:lll
```

Comment:
we shouldn't ignore the linter unless its actually necessary.  In this case we can break the long line up over multiple lines.  We allow it to go up to 140 which should be plenty.  

Same comment on line 30